### PR TITLE
Correction of overlays dimensions

### PR DESCRIPTION
**Pull request by swistach**
### Context
Currently, after initialization, overlays' elements have incorrect dimensions.
In this PR I try to adjust these elements to their content by removing unnecessary space eg.: 
- additional pixels for `box-shadow`,
- additional space for scrollbars

### How has this been tested?
1. Initialize Handsontable with fixedRowsTop, fixedRowsBottom and fixedColumnsLeft.
2. Measure overlay's dimension using devtools

### Types of changes
- [x] Bugfix (a non-breaking change which fixes an issue)

### Related issue(s):
1. #3873

### Checklist:
- [x] My change requires a change to the documentation.

### CODE REVIEW

**Comment by wojciechczerniak** [comment]
⚠️ Note: After this PR it's necessary to update styles for fixed overlays in our documentation.

File: src/3rdparty/walkontable/src/overlay/bottom.js [hunk 1]
```diff
@@ -207,22 +207,18 @@ class BottomOverlay extends Overlay {
...
 
     this.clone.wtTable.hider.style.width = this.hider.style.width;
-    this.clone.wtTable.holder.style.width = this.clone.wtTable.holder.parentNode.style.width;
-
-    if (scrollbarWidth === 0) {
-      scrollbarWidth = 30;
-    }
-    this.clone.wtTable.holder.style.height = `${parseInt(this.clone.wtTable.holder.parentNode.style.height, 10) + scrollbarWidth}px`;
+    holder.style.width = holder.parentNode.style.width;
+    holder.style.height = `${parseInt(holder.parentNode.style.height, 10)}px`;
```
Comment (budnix) [inline src/3rdparty/walkontable/src/overlay/bottom.js]:
```suggestion
    holder.style.height = holder.parentNode.style.height;
```
Correct me if I'm wrong but if I understand correctly this line can be simplified by reassigning the height of the parent element directly. Without reparsing from string -> number -> string?

After the changes, the tests still pass.

File: src/3rdparty/walkontable/src/overlay/left.js [hunk 2]
```diff
@@ -180,22 +180,18 @@ class LeftOverlay extends Overlay {
...
 
     this.clone.wtTable.hider.style.height = this.hider.style.height;
-    this.clone.wtTable.holder.style.height = this.clone.wtTable.holder.parentNode.style.height;
-
-    if (scrollbarWidth === 0) {
-      scrollbarWidth = 30;
-    }
-    this.clone.wtTable.holder.style.width = `${parseInt(this.clone.wtTable.holder.parentNode.style.width, 10) + scrollbarWidth}px`;
+    holder.style.height = holder.parentNode.style.height;
+    holder.style.width = `${parseInt(holder.parentNode.style.width, 10)}px`;
```
Comment (budnix) [inline src/3rdparty/walkontable/src/overlay/left.js]:
```suggestion
    holder.style.width = holder.parentNode.style.width;
```

File: src/3rdparty/walkontable/src/overlay/top.js [hunk 3]
```diff
@@ -188,22 +188,18 @@ class TopOverlay extends Overlay {
...
 
     this.clone.wtTable.hider.style.width = this.hider.style.width;
-    this.clone.wtTable.holder.style.width = this.clone.wtTable.holder.parentNode.style.width;
-
-    if (scrollbarWidth === 0) {
-      scrollbarWidth = 30;
-    }
-    this.clone.wtTable.holder.style.height = `${parseInt(this.clone.wtTable.holder.parentNode.style.height, 10) + scrollbarWidth}px`;
+    holder.style.width = holder.parentNode.style.width;
+    holder.style.height = `${parseInt(holder.parentNode.style.height, 10)}px`;
```
Comment (budnix) [inline src/3rdparty/walkontable/src/overlay/top.js]:
```suggestion
    holder.style.height = holder.parentNode.style.height;
```

**Comment by budnix** [comment]
Besides the suggestions which I proposed to this PR I haven't found any issues. As @wojciechczerniak said after merging this issue docs styles for a demo with fixed headers  https://handsontable.com/docs/demo-fixing.html has to be fixed too.

File: src/3rdparty/walkontable/src/overlay/bottom.js (hunk 1 above)
Comment (swistach) [reply src/3rdparty/walkontable/src/overlay/bottom.js]: You are right. It's even better. It will be exactly as it should be (eg.: 17.4px). Thank you :+1:

**Comment by swistach** [comment]
The added issue to docs' repository: handsontable/docs#87

**Pull request merged**